app/views/reports/regions/diabetes/_bs_below_200_card.html.erb

```diff
@@ -0,0 +1,80 @@
+<% denominator_copy = with_ltfu ? "diabetes_denominator_with_ltfu_copy" : "diabetes_denominator_copy" %>
+<% data = pivot_period_data(@data.slice(:bs_below_200_rates, :bs_below_200_patients, :adjusted_diabetes_patient_counts)) %>
```

Comment:
Is this where we would [support the LTFU toggle](https://github.com/simpledotorg/simple-server/pull/3694/files#diff-1920189d420fe89ceec67f02ee365f2586073d12a5716d252b450820a8555712R766-R773)when we get to it?